package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "private": true,
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack --config webpack.config.js --watch"
```

Comment:
- [ ] [OPTIONAL] I'd advise you to the commands below as the "start" server command is currently not working.
  ![image](https://github.com/abiy006/Leaderboard/assets/104968236/0ffcb283-3f3b-40e5-b4a0-55550e0c05a1)